src/authenticate.cpp

```diff
@@ -65,6 +65,7 @@ Identity* authenticateWithId(SEXP con_, SEXP uuid_, SEXP ip_address_) {
 
     std::string uuid = Rcpp::as<std::string>(uuid_);
     std::string ip_address = Rcpp::as<std::string>(ip_address_);
+    bool isEmrsId = Rcpp::as<bool>(isEmrsId_);
```

Comment:
That is a side effect of us never having cleaned up / modernized the signature.  We could just use `std::striing` and `bool` there.  (Obviously not an action item for the PR.)